src/components/Avatar.js

```diff
@@ -5,5 +5,9 @@
 export default function Avatar(props) {    
     const { person } = props;
 
-    // return ...
+    return(
+        <div>
+            <img class="Avatar" src={person.imageUrl} alt="picture"></img>
```

Comment:
We use alt attribute in img element to make them accessible, they should have a meaningful value which represents the image